src/components/file-dropzone/file-dropzone.scss

```diff
@@ -0,0 +1,46 @@
+:host(limel-file-dropzone) {
+    position: relative;
+    display: grid;
+    position: relative;
+    height: 100%;
+    width: 100%;
```

Comment:
I think these two are unnecessary. It should be up-top the consumer to set a size on this element if they have to. But since this element is just a wrapper, it will inherit its size from their own children. They should just wrap them to provide functionality, and nothing more.